.buildkite/pipeline.yml

```
@@ -49,6 +50,16 @@ steps:
   - wait: ~
     continue_on_failure: true
 
+  - label: ":sonarqube: Continuous Code Inspection"
+    env:
+      VAULT_SONAR_TOKEN_PATH: "kv/ci-shared/platform-ingest/elastic/integrations/sonar-analyze-token"
+    agents:
+      image: "docker.elastic.co/cloud-ci/sonarqube/buildkite-scanner:latest"
+    command:
+      - "buildkite-agent artifact download build/test-coverage/coverage-*.out ."
```

Coverage files created by elastic-package are XML files
```suggestion
      - "buildkite-agent artifact download build/test-coverage/coverage-*.xml ."
```